### PR TITLE
#23 : Chnages to Refactoring class structure - output updated

### LINKED ISSUE #23: Refactoring `class` structure
I've had a play with the sample datafile.

I'd like to suggest these improvements, based on usability in OxygenXML please

* [ ] rename `refBody` to `body`
* [ ] rename `classProperties` to `summary` - keep it as first (compulsory) child of `body`
* [ ] move `signature`, `propulsion`, `remarks` to children of `refBody`, following  `summary`.  Items are optional, but in this specified order.
* [ ] remove XML comments, since they are visible in Oxygen (see below)

image

### PR DESCRIPTION
**Pull request by brahnavank**
Changes related to refactoring class structure and the new outupt updated.

Fixes #23

### CODE REVIEW

**Comment by IanMayo** [comment]
shouldn't the new DTDs use the new element name in the filename?

File: dita/regions/europe/france/unit_a_mock.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE class SYSTEM "../../../dtd/class.dtd">
```
Comment (IanMayo) [inline dita/regions/europe/france/unit_a_mock.xml:2]: IMHO this was a useful reference document during development, but we either don't need it now, or we need to store it somewhere else.  How about putting it in as a comment in the parent issue?

File: dita/regions/europe/france/unit_a.dita [hunk 2]
```diff
@@ -1,19 +1,87 @@
...
+            <property>
+                <proptype>Av TPK</proptype>
+                <propvalue>4.4</propvalue>
+            </property>
+        </summary>
+        <signatures>
+            <!-- extends section, allows free content in Phase 1. Constrained table in Phase 2 -->
+            <title>Signature</title>
+            <table>
+                <!-- in Phase 2 this will be replaced by `TonalTable` which just allows `Tonals` body element -->
```
[the remark this answers is not in the document]
Comment (brahnavank) [reply dita/regions/europe/france/unit_a.dita:36]: Yah sure.

File: dita/regions/europe/france/unit_a_mock.xml (hunk 1 above)
Comment (brahnavank) [reply dita/regions/europe/france/unit_a_mock.xml:2]: could you please tell is it ok to add it in readme.md file

Comment (brahnavank) [reply dita/regions/europe/france/unit_a_mock.xml]: removed

**Review by IanMayo — APPROVED** [review]

**Pull request merged**